osv/logs.py

```diff
@@ -17,30 +17,29 @@
 from google.cloud import logging as google_logging
 
 
-def setup_gcp_logging(service_name):
-  """Set up GCP logging and error reporting."""
+class _ErrorReportingFilter:
+  """
+  A logging filter that adds necessary json fields to error logs so that they
```

Comment:
Nit: Capitalism acronyms

```suggestion
  A logging filter that adds necessary JSON fields to error logs so that they
```